packages/react-composites/tests/browser/chat/app/serve.js

```diff
@@ -0,0 +1,4 @@
+const express = require('express');
```

Comment:
Maybe a "webpack devserver" will serve all our needs? So that we don't need to build the bundle then serve it in the server

Reply:
I need to run the server inline in nodejs code that behaves well with jest.
couldn't make webpack devserver to work that way.

Reply:
This looks like we're just using the express server as a static content server

We can probably create some routes on the webpackdevserver as well 

Could be a future modification